cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -188,6 +212,313 @@ void test_outlet_rename() {
         ;
     }
 
+    @Test
+    void test_rating_spec_id_uncontrolled() {
+        // Structure of test:
+        // 1) Create the Outlet
+        // 2) Retrieve the Outlet and assert that it has a null RatingSpecId
+        // 3) Retrieve the location group associated with the outlet
+        // 4) Modify the location group to have a rating spec id
+        // 5) Store the location group back to the db
+        // 6) Retrieve the Outlet and assert that it has the rating spec id
+        // 7) Delete the outlet
+        // 8) Delete the location group
+
+        String json = Formats.format(Formats.parseHeader(Formats.JSONV1, Outlet.class), NEW_RATED_OUTLET_UNCONTROLLED);
+
+        // Create the outlet
+        given()
+            .log().ifValidationFails(LogDetail.ALL, true)
+            .contentType(Formats.JSONV1)
+            .body(json)
+            .header(AUTH_HEADER, USER.toHeaderValue())
+            .queryParam(FAIL_IF_EXISTS, "false")
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .post("projects/outlets")
+        .then()
+            .log().ifValidationFails(LogDetail.ALL, true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_CREATED));
+
+        // Get the outlet, assert that it has null rating spec id
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
+            .accept(Formats.JSONV1)
+            .queryParam(Controllers.OFFICE, OFFICE_ID)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .get("projects/outlets/" + RATED_OUTLET_LOCATION_UNCONTROLLED.getName())
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_OK))
+            .body("rating-spec-id", equalTo(null))
```

Comment:
Use nullValue() instead of equalTo(null).

Reply:
Swapped